RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -2,33 +2,42 @@
 @using System.Globalization
 @inject IBillRepository billRepository
 
-@section Styles {
-<link rel="stylesheet" href="/css/BillDetails.css" />
-}
 @{
     var billId = Guid.Parse(RouteData.Values["guid"].ToString());
-    if (billRepository.GetById(billId) is null)
+    var currentBill = billRepository.GetById(billId);
+    if (currentBill is null)
     {
-        <div id="errorText" style="color:red"> Bill was not found!</div>
+        <div class="alert alert-danger text-center">
+            Bill was not found!
+        </div>
     }
     else
     {
-        var item = billRepository.GetById(billId);
-        <div class="bill-summary">
-            <div t-id="description">@item.Description</div>
-            <div t-id="personWhoPaid">Paid by: @(item.EntryLines[0].Person.FirstName)
-                @(item.EntryLines[0].Person.LastName) </div>
-            <div t-id="date">@item.Date.ToString("D")</div>
-            <div t-id="amount">Total amount: @item.EntryLines[0].Amount€</div>
+        <h1 class="text-center">Bill Details</h1>
+        var item = currentBill;
+        <div class="p-3 border rounded mx-auto" style ="background-color: #f7f7f7">
+            <dl class="row">
+                <dt class="col-12 col-sm-6">Description:</dt>
+                <dd t-id="description" class="col-12 col-sm-6 text-sm-end">@item.Description</dd>
```

Comment:
Something I don't know about Razor: if you enter `<b>something</b>` as description, does Razor render the HTML tags escaped or unescaped? Please add a ticket to do a HTML Injection review just to be certain.

Reply:
By default it should be escaped